pkg/installmanager/installmanager_test.go

```diff
@@ -237,16 +240,25 @@ func TestInstallManager(t *testing.T) {
 			// We don't want to run the uninstaller, so stub it out
 			im.cleanupFailedProvision = alwaysSucceedCleanupFailedProvision
 
+			// Save the list of actuators so that it can be restored at the end of this test
+			actuatorsSaved := actuators
+			actuators = []Actuator{&awsActuator{awsClientFn: func(c client.Client, secretName, namespace, region string, logger log.FieldLogger) (awsclient.Client, error) {
+				return mocks.mockAWSClient, nil
+			}}}
```

Comment:
This feels off, any time we have to do this in a test carries risk state leaks from one test to another, and in this case I can't see any major reason why we need to track the actuators we know about in a global variable.

How about dropping the global array, add an InstallManager.LogGatherer or Actuator field, and initialize it in NewInstallManagerCommand. Your tests here have direct access to the InstallManager type and are already manipulating fields on it to control what is tested, so you can just im.actuator = awsActuator or equivalent vars if they get renamed.